runtimes/pythonrt/build.go

```diff
@@ -93,10 +99,9 @@ func findExports(log *zap.Logger, fsys fs.FS) ([]sdktypes.BuildExport, error) {
 
 	var pyExe string
 	if !isUserPy {
-		if err := ensureVEnv(log, sysPyExe); err != nil {
+		if pyExe, err = ensureVEnv(log, envName, sysPyExe); err != nil {
```

Comment:
You need to pass the user requirements file (if they exist).

Reply:
done